xml/art-obs-beginners-guide.xml

```diff
@@ -175,24 +167,28 @@ https://www.suse.com/communities/blog/suse-studio-integration/
    <title>Projects and Packages</title>
    <para>
     In &obsa;, packages are organized in <emphasis>projects</emphasis>.
-    A single project can contain several packages and serves a specific
-    purpose, defines access control, other repositories, and build targets
-    (operating systems and architectures).
+    A single project can contain several packages and usually serves a specific
+    organizational purpose. Generic access control, related repositories, and
+    build targets (operating systems and architectures) are all defined at
+    the project level.
    </para>
    <para>
     Projects can also contain other projects (subprojects) to structure work.
     Each project name is separated by colons.
-    For example, all packages for fonts are collected in the project
-    <uri>M17N:fonts</uri> and all relevant packages for the Python programming
-    language are available in the <uri>devel:languages:python</uri> project.
+    For example, in the &opensuse; Build Service, packages for fonts are
+    collected in the project
+    <uri>M17N:fonts</uri> which is a subproject of <uri>M17N</uri>.
+    Packages for the Python programming
+    language are available in the <uri>devel:languages:python</uri> project
+    which is a subproject of <uri>devel:languages</uri> which itself is a
+    subproject of <uri>devel</uri>.
    </para>
    <para>
-    As a user, you will build your packages normally in your
+    As a user, you will normally build packages in your
     <firstterm baseform="Home Project">home project</firstterm>, available
```

Comment:
> Maybe we could just speak in plural "the users home projects" to include also the subprojects?

I would avoid the plural for home project. IMHO, I fear, that would confuse people, because there is basically only one home project. 😉

For your specific wording, I would prefer "the user's home project and his subprojects". Maybe we need to make clear that subprojects includes also branches (which is just a "special" subproject).